### PR TITLE
Checking the JSON RPC interface is working.

### PR DESCRIPTION
**Pull request by hackaugusto**
The blockchain service is no longer querying the blockchain for the
discovery address, so the previous message about the json rpc interface
was not working.

This adds an explicit function for checking if the JSON RPC interface is
enabled and a temporary check disabling the Parity client.

### CODE REVIEW

**Comment by hackaugusto** [comment]
Checking if the JSON RPC is working, if not warning the user https://github.com/raiden-network/raiden/issues/1065

File: raiden/ui/cli.py [hunk 1]
```diff
@@ -90,12 +97,61 @@ def toggle_trace_profiler(raiden):
         raiden.profiler.start()
 
 
+def check_json_rpc(client):
+    try:
+        client_version = client.call('web3_clientVersion')
+    except (requests.exceptions.ConnectionError, JSONRPCClientReplyError):
+        print(
+            "\n"
```
Comment (LefterisJP) [inline raiden/ui/cli.py]: SCan you change this and all other string literlas to be with single quotes?

Comment (hackaugusto) [reply raiden/ui/cli.py]: Humm, the next line actually uses a single quote, I would like to not quote that and keep these quotes consistent (all double quotes), do you really want to change it?

Comment (LefterisJP) [reply raiden/ui/cli.py]: no it's fine then.

**Pull request merged**